Model/Resolver/PlaceRazorpayOrder.php

```diff
@@ -0,0 +1,141 @@
+<?php
+declare (strict_types = 1);
+
+namespace Razorpay\Magento\Model\Resolver;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\QuoteGraphQl\Model\Cart\GetCartForUser;
+use Magento\Quote\Api\CartManagementInterface;
+use Razorpay\Magento\Model\PaymentMethod;
+
+class PlaceRazorpayOrder implements ResolverInterface
+{
+    /**
+     * @var \Magento\Framework\App\Config\ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * @var \Magento\Quote\Api\CartManagementInterface
+     */
+    protected $cartManagement;
+
+    /**
+     * @var \Magento\Framework\App\ObjectManager
+     */
+    protected $_objectManager;
+
+    /**
+     * @var \Magento\Sales\Api\Data\OrderInterface
+     */
+    protected $order;
+
+    /**
+     * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+     * @param GetCartForUser $getCartForUser
+     * @param \Magento\Quote\Api\CartManagementInterface $cartManagement
+     * @param PaymentMethod $paymentMethod
+     * @param \Magento\Sales\Api\Data\OrderInterface $order
+     */
+    public function __construct(
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
+        GetCartForUser $getCartForUser,
+        \Magento\Quote\Api\CartManagementInterface $cartManagement,
+        PaymentMethod $paymentMethod,
+        \Magento\Sales\Api\Data\OrderInterface $order
+    )
+    {
+        $this->scopeConfig    = $scopeConfig;
+        $this->getCartForUser = $getCartForUser;
+        $this->cartManagement = $cartManagement;
+        $this->rzp            = $paymentMethod->rzp;
+        $this->_objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+        $this->order          = $order;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
+    {
+        if (empty($args['order_id']))
+        {
+            throw new GraphQlInputException(__('Required parameter "order_id" is missing'));
+        }
+        try
+        {
+            $storeScope = \Magento\Store\Model\ScopeInterface::SCOPE_STORE;
+            $storeId = (int) $context->getExtensionAttributes()->getStore()->getId();
+            $receipt_id        = $args['order_id'];
+
+            $collection = $this->_objectManager->get(\Magento\Sales\Model\Order::class)
+            ->getCollection()
+            ->addFieldToSelect('*')
```

Comment:
@enishant , can we apply similar changes here as well.

Reply:
@ramth05 Updated order query